docs/content/development/experimental.md

```diff
@@ -12,3 +12,31 @@ druid.extensions.coordinates=["io.druid.extensions:druid-histogram:{VERSION}"]
 ```
 
 The configuration for all the indexer and query nodes need to be updated with this.
+
+
+#### KafkaSimpleConsumerFirehose
```

Comment:
can we create a separate doc for this firehose as we did for the other experimental features?